sdk/communication/azure-communication-sms/src/test/java/com/azure/communication/sms/CommunicationLoggerPolicy.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.sms;
+
+import com.azure.core.http.HttpPipelineCallContext;
+import com.azure.core.http.HttpPipelineNextPolicy;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.util.logging.ClientLogger;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * A policy that logs the Http Response
+ */
+
+public class CommunicationLoggerPolicy implements HttpPipelinePolicy {
```

Comment:
Please follow this PR to remove the CommunicationLoggerPolicy: https://github.com/Azure/azure-sdk-for-java/pull/19318